### PR TITLE
Remove the ability to addlog in addCommand

### LINKED ISSUE #131: add command still uses l/date|entry

### PR DESCRIPTION
**Pull request by zhiyi12345**
closes #131

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -38,8 +38,7 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney "
-            + PREFIX_LOG + "20 Jan 2022|First visit";
+            + PREFIX_TAG + "owesMoney ";
```
Comment (slidings) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:41]: i think this for the tag example a bit weird to label a patient as owesmoney, can consider changing in next iteration

File: src/main/java/seedu/address/model/person/Person.java [hunk 2]
```diff
@@ -43,6 +53,21 @@ public Person(Name name, IdentityNumber identityNumber, Phone phone, Email email
...
+     * @param name The person's name.
+     * @param identityNumber The person's identity number.
+     * @param phone The person's phone number.
+     * @param email The person's email address.
+     * @param address The person's physical address.
+     * @param tags The set of tags associated with the person.
+     * @throws NullPointerException if any of the arguments are null.
+     */
+    public Person(Name name, IdentityNumber identityNumber, Phone phone, Email email, Address address, Set<Tag> tags) {
+        this(name, identityNumber, phone, email, address, tags, new HashSet<>());
```
Comment (slidings) [inline src/main/java/seedu/address/model/person/Person.java:69]: looks good

**Review by slidings — APPROVED** [review]

File: src/main/java/seedu/address/model/person/Person.java [hunk 3]
```diff
@@ -30,7 +30,17 @@ public class Person {
...
+     * @param identityNumber The person's identity number.
+     * @param phone The person's phone number.
+     * @param email The person's email address.
+     * @param address The person's physical address.
+     * @param tags The set of tags associated with the person.
+     * @param logs The set of logs associated with the person.
+     * @throws NullPointerException if any of the arguments are null.
      */
     public Person(Name name, IdentityNumber identityNumber, Phone phone, Email email, Address address, Set<Tag> tags,
                   Set<Log> logs) {
```
Comment (jeanyang04) [inline src/main/java/seedu/address/model/person/Person.java:46]: I like how you added new constructors for Person class, making it more flexible.

**Review by jeanyang04 — COMMENTED** [review]
All good!

**Pull request merged**